### DISCUSSION TITLE
Website Optimizations

### DISCUSSION THREAD

**Discussion by fmerian** [dubinc/dub#458] [General]
hey!

I enjoyed contributing to Dub's marketing site (#200, #384, #403). 
with the [recent migration to Turborepo](https://dub.co/blog/turborepo-migration), is it still possible to contribute to the website, blog and changelog?

thanks, keep it up @steven-tey!

**Comment by steven-tey** [comment]
Thank you for your previous contributions @fmerian! Unfortunately this is no longer possible since the website (dub.co) is now private 😅

**Reply by fmerian** [reply]
@steven-tey any plans to make it public again with the new org? 😇
